### PR TITLE
Core update

### PR DESCRIPTION
**Pull request by yashovardhan**
Updates according to new released web3auth version. Spent quite some time figuring out how to display version in docusaurus, but to no avail. We need to think about a way to document the latest version of each sdk

### CODE REVIEW

File: src/common/guides/_openlogin-config-code-discord.mdx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ```js
 const openloginAdapter = new OpenloginAdapter({
   adapterSettings: {
-    clientId: "YOUR-WEB3AUTH-CLIENT-ID",
+    clientId: "YOUR-WEB3AUTH-CLIENT-ID", //Optional - Provider only if you haven't provided it in the Web3Auth Instantiation Code
```
Comment (shahbaz17) [inline src/common/guides/_openlogin-config-code-discord.mdx:4]: While instantiating, it's a required field. Right? If so, we can remove this comment.

File: docs/sdk/web/adapters/torus-solana.mdx [hunk 2]
```diff
@@ -108,6 +108,7 @@ If you do pass `chainConfig` in an Adapter, it overwrites the `chainConfig` pass
 import { SolanaWalletAdapter } from "@web3auth/torus-solana-adapter";
 const solanaWalletAdapter = new SolanaWalletAdapter({
   adapterSettings: {
+    clientId: "YOUR_CLIENT_ID",
```
Comment (shahbaz17) [inline docs/sdk/web/adapters/torus-solana.mdx:111]: Let's change these with `YOUR_WEB3AUTH_CLIENT_ID` to be consistent in docs. There are other client id's as well such as from different providers.

File: src/common/guides/_openlogin-config-code-discord.mdx (hunk 1 above)
Comment (shahbaz17) [reply src/common/guides/_openlogin-config-code-discord.mdx:4]: Same applies to other references of this.

Comment (shahbaz17) [reply src/common/guides/_openlogin-config-code-discord.mdx:4]: Looking at https://github.com/Web3Auth/Web3Auth/blob/master/packages/core/src/core.ts#L31 looks like it's a required field.

**Review by shahbaz17 — CHANGES_REQUESTED** [review]

**Review by shahbaz17 — APPROVED** [review]
LGTM.

File: src/common/sdk/web/_openlogin-options.mdx [hunk 3]
```diff
@@ -15,7 +15,7 @@ import Tabs from "@theme/Tabs";
...
-| `clientId`              | `string`                              | Client ID for web3auth. Obtain your `clientId` from the [Web3Auth Developer Dashboard](https://dashboard.web3auth.io/).                                                                                                                                                                                                                                   | Yes      | N/A           |
+| `clientId?`             | `string`                              | Client ID for web3auth. Obtain your `clientId` from the [Web3Auth Developer Dashboard](https://dashboard.web3auth.io/).                                                                                                                                                                                                                                   | No       | N/A           |
```
Comment (himanshuchawla009) [inline src/common/sdk/web/_openlogin-options.mdx:18]: can you add same note here, that is it required here if not being passed in web3auth constructor.

File: src/common/sdk/web/_web3authcore-options.mdx [hunk 4]
```diff
@@ -11,18 +11,25 @@ import Tabs from "@theme/Tabs";
...
+| Parameter        | Type                                                                         | Description                                                                                                             | Optional                   | Default |
+| ---------------- | ---------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------- | -------------------------- | ------- |
+| `chainConfig`    | Partial `<CustomChainConfig>` & Pick `<CustomChainConfig, "chainNamespace">` | Custom configuration for your preferred blockchain. Read more about it [here](#chainconfig).                            | No                         |
+| `clientId`       | `string`                                                                     | Client ID for web3auth. Obtain your `clientId` from the [Web3Auth Developer Dashboard](https://dashboard.web3auth.io/). | No / Optional in localhost |
```
Comment (himanshuchawla009) [inline src/common/sdk/web/_web3authcore-options.mdx:17]: it should be YES (optional in localhost)

**Review by himanshuchawla009 — CHANGES_REQUESTED** [review]

File: src/common/sdk/web/_openlogin-options.mdx (hunk 3 above)
Comment (yashovardhan) [reply src/common/sdk/web/_openlogin-options.mdx:18]: done

**Pull request merged**